src/dom-builder.ts

```diff
@@ -345,8 +345,12 @@ class DomComponent<O, P, A> extends Component<O & P, A & P> {
     }
   }
   run(parent: Node, destroyed: Future<boolean>): Out<O & P, A & P> {
-    const elm = document.createElement(this.tagName);
-
+    const ns: string | undefined = (this.props as any).namespace
+    const elm: HTMLElement = ns 
+      ? (document.createElementNS(ns, this.tagName) as HTMLElement)
+      : document.createElement(this.tagName);
+    delete (this.props as any).namespace
```

Comment:
Why the `delete` here? I don't see why it is necessary and I actually think it may break things. A component can be run many times. And if the first invocation to `run` deletes the namespace then I won't be there for future runs of the component. In other words, `run` can have side-effects but it shouldn't mutate the component it is called on.

Reply:
Ah. I see. I think that can be fixed by adding `"namespace"` to the set `propKeywords` [here](https://github.com/funkia/turbine/blob/b17f1bc6e30512a3ab9a7dc14eb6ab5efa7b82d7/src/dom-builder.ts#L247).